src/pages/EntityDetails/Model/ApplicationsTab/ApplicationsTab.tsx

```diff
@@ -266,32 +128,32 @@ export default function ApplicationsTab({ filterQuery }: Props) {
   const getContentReveals = () => {
     return (
       <>
-        {!!appOffersTableLength && (
+        {!!localAppTableLength && (
```

Comment:
I rearranged these tables so that the local apps table is first so that the search box is at the top and also so that the content doesn't jump around as much when searching if there are multiple tables.